samplesheets/tests/test_views_taskflow.py

```diff
@@ -1090,9 +1091,14 @@ def setUp(self):
         self.assay_path = self.irods_backend.get_path(self.assay)
         self.path = os.path.join(self.assay_path, TEST_FILE_NAME)
         self.path_md5 = os.path.join(self.assay_path, f'{TEST_FILE_NAME}.md5')
+        # Second file
+        self.path2 = os.path.join(self.assay_path, TEST_FILE_NAME2)
+        self.path2_md5 = os.path.join(self.assay_path, f'{TEST_FILE_NAME2}.md5')
         # Create objects
         self.file_obj = self.irods.data_objects.create(self.path)
         self.md5_obj = self.irods.data_objects.create(self.path_md5)
+        self.file_obj2 = self.irods.data_objects.create(self.path2)
```

Comment:
Are `self.file_obj2` and `self.md5_obj2` actually used anywhere?

Reply:
Yep, with help of them the `self.path2` and `self.path2_md5` are being formed (and later in testing Batch views being used)